scenarios/happypath/configbranch/configbranch_change.go

```diff
@@ -0,0 +1,164 @@
+package configbranch
+
+import (
+	"strings"
+
+	apiclient "github.com/equinor/radix-cicd-canary/generated-client/client/application"
+	models "github.com/equinor/radix-cicd-canary/generated-client/models"
+	"github.com/equinor/radix-cicd-canary/scenarios/utils/array"
+	"github.com/equinor/radix-cicd-canary/scenarios/utils/config"
+	"github.com/equinor/radix-cicd-canary/scenarios/utils/env"
+	envUtil "github.com/equinor/radix-cicd-canary/scenarios/utils/env"
+	httpUtils "github.com/equinor/radix-cicd-canary/scenarios/utils/http"
+	"github.com/equinor/radix-cicd-canary/scenarios/utils/job"
+	"github.com/equinor/radix-cicd-canary/scenarios/utils/test"
+	log "github.com/sirupsen/logrus"
+)
+
+var logger *log.Entry
+
+type expectedStep struct {
+	name       string
+	components []string
+}
+
+// Change Tests that radixconfig is read from the branch defined as configBanch
+func Change(env envUtil.Env, suiteName string) (bool, error) {
+	logger = log.WithFields(log.Fields{"Suite": suiteName})
+
+	// Trigger build via web hook
+	ok := httpUtils.TriggerWebhookPush(env, config.App4ConfigBranch, config.App4CommitID, config.App4SSHRepository, config.App4SharedSecret)
+	if !ok {
+		return false, nil
+	}
+	logger.Infof("First job was triggered")
+
+	// Get first job
+	ok, jobSummary := test.WaitForCheckFuncOrTimeout(env, func(env envUtil.Env) (bool, interface{}) {
+		return job.IsListedWithStatus(env, config.App4Name, "Running")
+	})
+
+	if !ok {
+		log.Errorf("Could not get listed job for application %s status \"%s\" - exiting.", config.App4Name, "Running")
+		return false, nil
```

Comment:
I suggest just return error as it is done `scenarios/happypath/buildsecrets/buildsecrets_set.go` - it will be logged on the level above, I assume